homeassistant/components/fing/strings.json

```
@@ -0,0 +1,31 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "title": "Fing agent setup",
+        "data": {
+          "ip_address": "[%key:common::config_flow::data::ip%]",
+          "port": "[%key:common::config_flow::data::port%]",
+          "api_key": "[%key:common::config_flow::data::api_key%]"
+        },
+        "data_description": {
+          "ip_address": "IP address of the Fing Agent.",
+          "port": "Port number of the Fing API.",
+          "api_key": "API key used to authenticate with the Fing API."
+        }
+      }
+    },
+    "error": {
+      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]",
+      "invalid_api_key": "[%key:common::config_flow::error::invalid_api_key%]",
+      "timeout_connect": "[%key:common::config_flow::error::timeout_connect%]",
+      "unexpected_error": "[%key:common::config_flow::error::unknown]",
+      "url_error": "[%key:common::config_flow::error::invalid_host]",
```

The "%" chars are missing here, too:

```suggestion
      "unexpected_error": "[%key:common::config_flow::error::unknown%]",
      "url_error": "[%key:common::config_flow::error::invalid_host%]",
```